main.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
- It is not a good practice to disable linters. They're there to help you write a better/cleaner code

Reply:
Thanks for the tip @GzimAsani :100: